Gopkg.toml

```diff
@@ -31,3 +31,12 @@
 [[constraint]]
   name = "github.com/onsi/gomega"
   version = "=1.1.0"
+
+[prune]
+  non-go = true
+  go-tests = true
+  unused-packages = true
```

Comment:
@jodh-intel `prune` was already part of the Gopkg.toml. Thanks

Reply:
@jodh-intel changes applied to `bundle.go`. thanks

Reply:
Hi @GabyCT - this doesn't appear to be working though as the pruning is supposed to remove all vendored packages tests for example. Can you try re-running `dep ensure && dep prune`?

Reply:
@jodh-intel sorry I am little confused, what do you mean that prunning is supposed to remove all vendored package tests? I see that when I ran the `dep prune` , it is removing around 118 files with *_test.go. Thanks